obs-packaging/create-pkg-branch.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+[ -z "${DEBUG}" ] || set -o xtrace
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir=$( cd $( dirname "${BASH_SOURCE[0]}" ) && pwd )
+script_name="$(basename "${BASH_SOURCE[0]}")"
+
+source "${script_dir}/../scripts/lib.sh"
+
+readonly home_project="home:katacontainers"
+readonly template_pkg="kata-pkg-template"
+arch_target=$(uname -m)
+
+projects=(
+qemu-lite
+qemu-vanilla
+linux-container
+kata-containers-image
+proxy
+shim
+ksm-throttler
+runtime
+)
+
+pkg_exist(){
+	local project="$1"
+	local pkg="$2"
+
+	osc list "${project}" | grep "${pkg}" || return 1
+	return 0
+
+}
+
+# Array of repositories.
+#
+# Each element is comprised of mutiple parts in the form:
+#
+#   name::project::repository
+#
+typeset -a repos
+repos+=(CentOS_7::CentOS:CentOS-7::standard)
+repos+=(Fedora_26::Fedora:26::standard)
+repos+=(Fedora_27::Fedora:27::standard)
+repos+=(Fedora_28::Fedora:28::standard)
+repos+=(RHEL_7::RedHat:RHEL-7::standard)
+repos+=(SLE_12_SP3::SUSE:SLE-12-SP3:GA::standard)
+repos+=(openSUSE_Leap_42.3::openSUSE:Leap:42.3::standard)
+repos+=(xUbuntu_16.04::Ubuntu:16.04::universe)
+repos+=(xUbuntu_17.10::Ubuntu:17.10::universe)
+repos+=(xUbuntu_18.04::Ubuntu:18.04::universe)
+
+# Array of mantainers.
+#
+# Each element is comprised of mutiple parts in the form:
+#
+#   userid::role
+#
+typeset -a mantainers
+mantainers+=(jcvenega::maintainer)
+
+create_repos_xml_nodes(){
+	for entry in "${repos[@]}"
+	do
+		[ -z "$entry" ] && die "found empty entry"
+		local name=$(echo "$entry"| awk -F"::" '{print $1;}')
+		local project=$(echo "$entry"| awk -F"::" '{print $2;}')
+		local repository=$(echo "$entry"| awk -F"::" '{print $3;}')
+		[ -z "$name" ] && die "no name for entry '$entry'"
+		[ -z "$project" ] && die "no project for entry '$entry'"
+		[ -z "$repository" ] && die "no repository for entry '$entry'"
+		echo "  <repository name=\"${name}\">"
```

Comment:
This is fine, but it might be clearer to change all the functions that create XML so they write the data out using a here doc and `cat` (as you can drop all the backslashes) and the "shape" of the document/snippet is much easier to see:

```
cat <<EOT
<repository name="${name}">
    <path project="${project}" repository="${repository}"/>
    :

EOT
```